docs/docs/infrahubctl/infrahubctl-repository.mdx

```diff
@@ -47,6 +48,26 @@ $ infrahubctl repository add [OPTIONS] NAME LOCATION
 * `--config-file TEXT`: [env var: INFRAHUBCTL_CONFIG; default: infrahubctl.toml]
 * `--help`: Show this message and exit.
 
+## `infrahubctl repository init`
+
+Initialize a new Infrahub repository.
+
+**Usage**:
+
+```console
+$ infrahubctl repository init [OPTIONS] DST
+```
+
+**Arguments**:
+
+* `DST`: [required]
+
+**Options**:
+
+* `--data PATH`
```

Comment:
Same, what is data in this case? A Path to a file? To what file? To a directory?

Reply:
I have added helpers to the arguments for the docs to be better.